src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -190,17 +190,46 @@ public static class StandardLearnersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer.
+        /// Predict a target using a linear binary classification model trained with averaged perceptron trainer.
         /// </summary>
+        /// <remarks>
+        /// Perceptron is a classification algorithm that makes its predictions by finding a separating hyperplane.
```

Comment:
>Perceptron  [](start = 12, length = 11)

The Perceptron

Reply:
done

---
In reply to: [256222592](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256222592) [](ancestors = 256222592)